backend/Node/routes/openWeatherRoute.ts

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import { currentWeatherData } from './route-helpers/openweather-helpers';
+import { isLatitude, isLongitude } from './route-helpers/mapbox-helpers';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    res.send("Open Weather API");
+});
+
+router.get('/reverseWeather/:latLon', async (req, res) => {
+    const latLon = req.params.latLon.split(',');
+    if (!isLatitude(latLon[0]) || !isLongitude(latLon[1])) {
+        res.status(400).send("Invalid lat/lon value(s)");
+    }
+    const weatherData = await currentWeatherData(latLon[0], latLon[1]);
+
+    const relevantData = [];
+    relevantData.push(
```

Comment:
Is there a reason we want this as an array? Would it make more sense to use an object (dictionary if Luke is reading this)? 

That way, whoever calls this endpoint won't have to know the order which the values are returned.

Reply:
Yea I think it'd be a good idea to process the weather data results (if they need more processing) here and then send back a dictionary / json. I think we also agreed I only needed the feels_like field instead of temp, so there's no need to send temp back 😄

Reply:
Oh in Ireena's mocks we wanna display both `feels_like` and `temp` in the app. We just won't need to send 2 values to the ML server

Reply:
Good point! I'll change it to an object so its easier to process, and yeah I think we mentioned that we'll send both `feels_like` and `temp` and then the ML server would just work with `feels_like`?